english/ponder/ponder-about-life.md

```diff
@@ -1,5 +1,5 @@
 You start thinking about how you ended up this bizzare place. Could the marshallmallow wall represent something
-significant? Maybe the lighter represents life, where as fork represents survival. You take another look around 
```

Comment:
Corrected previous lines, and put back line-breaks where they were. So they are not modified. Please review @casey-collab .